lambda/solarwinds-apm/wrapper

```diff
@@ -27,16 +27,6 @@ export LAMBDA_LAYER_PKGS_DIR="/opt/python";
 export PYTHONPATH="$LAMBDA_LAYER_PKGS_DIR:$PYTHONPATH";
 export PYTHONPATH="$LAMBDA_RUNTIME_DIR:$PYTHONPATH";
 
-# Default opt into OTLP metrics export in lambda
-if [ -z "${SW_APM_EXPORT_METRICS_ENABLED}" ]; then
-    export SW_APM_EXPORT_METRICS_ENABLED=true
-fi
```

Comment:
This should have been removed in a different PR where `true` is now the default.